### PR TITLE
Add Aptos docs

### PR DESCRIPTION
**Pull request by ginstagram**
## Description

Add Aptos docs to Quickstart, Platform, Operations, Tutorials, and Blockchains.
Update config.js respectively

### CODE REVIEW

File: docs/operations/aptos/tools.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+See also [View node access and credentials](/platform/view-node-access-and-credentials).
+
+### REST API
+
+Interact with your Aptos node using the [Aptos Node API](https://fullnode.devnet.aptoslabs.com/v1/spec#/).
+
+Use your Chainstack Aptos REST endpoint:
+
+```sh
+https://nd-981-674-322.int.chainstack.com/v1/blocks/by_height/{block_height}
```
Comment (akegaviar) [inline docs/operations/aptos/tools.md:52]: Change to the dummy

File: docs/operations/aptos/tools.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+Example to get block information by block height:
+
+<CodeSwitcher :languages="{kp:'Key-protected',pp:'Password-protected'}">
+<template v-slot:kp>
+
+```sh
+curl --request GET \
+  --url https://nd-981-674-322.int.chainstack.com/v1/blocks/by_height/2047048\
```
Comment (akegaviar) [inline docs/operations/aptos/tools.md:62]: Change to dummy

File: docs/operations/aptos/tools.md (hunk 1 above)
Comment (soos3d) [reply docs/operations/aptos/tools.md:52]: This is already a fake endpoint, but let's update it if we have a dummy that we use to keep consistency.

Comment (ginstagram) [reply docs/operations/aptos/tools.md]: done

File: docs/operations/aptos/tools.md (hunk 2 above)
Comment (ginstagram) [reply docs/operations/aptos/tools.md]: done

File: docs/blockchains/aptos.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+meta:
+  - name: description
+    content: Aptos is a layer 1 novel chain created using Move, which is a Rust-based computer programming language.
+  - name: keywords
+    content: aptos blockchain move pos
+---
+
+# Aptos
+
+Aptos is a permissionless non-EVM compatible layer 1 novel chain created using [Move](https://move-language.github.io/move/), which is a Rust-based computer programming language.
```
Comment (akegaviar) [inline docs/blockchains/aptos.md:11]: Let's drop `novel`, I don't think it fits here.

Comment (akegaviar) [reply docs/blockchains/aptos.md:11]: > created using [Move](https://move-language.github.io/move/)

I don't think it's created using Move. It utilizes/uses Move/

File: docs/blockchains/aptos.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+meta:
+  - name: description
+    content: Aptos is a layer 1 novel chain created using Move, which is a Rust-based computer programming language.
```
Comment (akegaviar) [inline docs/blockchains/aptos.md:4]: Drop `computer`.

File: docs/tutorials/aptos/publish-module-to-save-and-retrieve-message-on-aptos.md [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+meta:
+  - name: description
+    content: Learn how to publish a module to save and retrieve a message on Aptos
+  - name: keywords
+    content: tutorial aptos message move module
+---
+
+# Publish a module to save and retrieve a message on Aptos
+
+Aptos uses its own terminology for widely-known web3 entities. Smart contracts are called *Modules* and are written in the [Move language](https://move-language.github.io/move/). Modules are also not deployed but *published* on the Aptos chain.
```
Comment (akegaviar) [inline docs/tutorials/aptos/publish-module-to-save-and-retrieve-message-on-aptos.md:11]: Across all our assets, we use `web3` and `Web` interchangeably. Let's settle on [Web3 capitalized](https://en.wikipedia.org/wiki/Web3), add it to our style guide and make sure we use `Web3` across all assets.

File: docs/tutorials/aptos/publish-module-to-save-and-retrieve-message-on-aptos.md [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+1. With Chainstack, access your [Aptos node credentials](/platform/view-node-access-and-credentials).
+1. Set up your Martian wallet to work through the Chainstack Aptos node.
+1. Fund your account through the [Aptos testnet faucet](https://aptoslabs.com/testnet-faucet).
+1. Install the [Aptos CLI](https://aptos.dev/cli-tools/aptos-cli-tool/install-aptos-cli).
+1. Create and configure your Aptos project.
+1. Create a Move project inside your Aptos project.
+1. Create a module in the Move language.
+1. Compile and test the Move module.
+1. Publish the Move module.
+1. Save and retrieve a message on the Aptos chain
```
Comment (akegaviar) [inline docs/tutorials/aptos/publish-module-to-save-and-retrieve-message-on-aptos.md:43]: Full stop?

File: docs/tutorials/aptos/publish-module-to-save-and-retrieve-message-on-aptos.md [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+
+### Install the Aptos CLI
+
+You need the Aptos CLI to interact with your Move module. Set up the [Aptos CLI](https://aptos.dev/cli-tools/aptos-cli-tool/install-aptos-cli).
+
+### Create and configure an Aptos project
+
+1. In your project directory, run `aptos init` > `custom`.
+   This will start a configuration process, during which you need to set up your Chainstack endpoint and Martian wallet private key. Adding the private key will retrieve your Aptos public address automatically.
+1. Add your [Chainstack endpoint](/platform/view-node-access-and-credentials). Paste your Aptos node endpoint deployed with Chainstack, for example: `https://nd-123-456-789.int.chainstack.com/`.
```
Comment (akegaviar) [inline docs/tutorials/aptos/publish-module-to-save-and-retrieve-message-on-aptos.md:75]: > int.chainstack.com

This is dev, don't use it in public docs. For public docs, it's `p2pify`, same as prod

**Review by akegaviar — CHANGES_REQUESTED** [review]
Changes please

**Review by akegaviar — DISMISSED** [review]

File: docs/blockchains/aptos.md (hunk 3 above)
Comment (conor2e) [reply docs/blockchains/aptos.md]: I used the word novel as it was referred to in [this article](https://coinmarketcap.com/alexandria/article/what-is-aptos-the-ultimate-guide-to-the-aptos-ecosystem), but I'll remove it now.

Comment (conor2e) [reply docs/blockchains/aptos.md]: i referred to [this article](https://news.bitcoin.com/vc-backed-blockchain-project-aptos-reveals-protocols-mainnet-has-arrived/) for Move and went with "used" as opposed to more colourful terms like "crafted by", but will change to "uses" as requested.

Comment (conor2e) [reply docs/blockchains/aptos.md]: I have changed the sentence to the following: "Aptos is a permissionless non-EVM compatible layer 1 chain created by utilizing Move, which is a Rust-based computer programming language."

Comment (conor2e) [reply docs/blockchains/aptos.md]: Nevermind, Lera made all the requested changes already.

**Review by akegaviar — DISMISSED** [review]

**Review by akegaviar — APPROVED** [review]

**Review by cthtuf — APPROVED** [review]

**Review by Maxitosh — APPROVED** [review]

**Pull request merged**